src/webrtc/vad.md

```diff
@@ -0,0 +1,76 @@
+# Active Speaker Detection
```

Comment:
Could we start a new sentence in a new line? It's easier to see changes in subsequent PRs. The other thing is enumeration, you can simply write:

```
1.
1.
1.
1.
```

Reply:
> The other thing is enumeration, you can simply write:

I know, but I don't like this style of enumeration. And I don't think I'm alone in that feeling, I have never seen a Markdown editor that doesn't increment enumeration.